audio_service/example/lib/main.dart

```diff
@@ -151,81 +360,124 @@ class MediaState {
   MediaState(this.mediaItem, this.position);
 }
 
-/// An [AudioHandler] for playing a single item.
-class AudioPlayerHandler extends BaseAudioHandler with SeekHandler {
-  static final _item = MediaItem(
-    id: 'https://s3.amazonaws.com/scifri-episodes/scifri20181123-episode.mp3',
-    album: "Science Friday",
-    title: "A Salute To Head-Scratching Science",
-    artist: "Science Friday and WNYC Studios",
-    duration: const Duration(milliseconds: 5739820),
-    artUri: Uri.parse(
-        'https://media.wnyc.org/i/1400/1400/l/80/1/ScienceFriday_WNYCStudios_1400.jpg'),
-  );
+class SeekBar extends StatefulWidget {
```

Comment:
Did you change anything in SeekBar? If not, can you restructure the PR so that the diffs only show what you changed, here and elsewhere?

Reply:
Done in new PR